src/static/riot/competitions/detail/participant_manager.tag

```diff
@@ -0,0 +1,108 @@
+<participant-manager>
+    <div show="{participants}">
+        <div class="ui icon input">
+            <input type="text" placeholder="Search..." ref="participant_search" onkeyup="{ search_participants }">
+            <i class="search icon"></i>
+        </div>
+        <select ref="participant_status" class="ui dropdown" onchange="{ update_participants.bind(this, undefined) }">
+            <option value="">Status</option>
+            <option value="-">----</option>
+            <option value="approved">Approved</option>
+            <option value="pending">Pending</option>
+            <option value="denied">Denied</option>
+            <option value="unknown">Unknown</option>
+        </select>
+        <table class="ui celled striped table">
+            <thead>
+            <tr>
+                <th>Username</th>
+                <th>Email</th>
+                <th>Status</th>
+                <th class="center aligned">Actions</th>
+            </tr>
+            </thead>
+            <tbody>
+            <tr each="{participants}">
+                <td>{username}</td>
+                <td>{email}</td>
+                <td>{_.startCase(status)}</td>
+                <td class="right aligned">
+                    <button class="mini ui red button icon" show="{status !== 'denied'}"
```

Comment:
would prefer all html attributes line up, , so `show=` should be on its own line lining up with `class=` as well